staging/src/k8s.io/pod-security-admission/admission/api/validation/validation.go

```diff
@@ -18,11 +18,75 @@ package validation
 
 import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
-	"k8s.io/pod-security-admission/admission/api"
+	admissionapi "k8s.io/pod-security-admission/admission/api"
+	"k8s.io/pod-security-admission/api"
 )
 
-func ValidatePodSecurityConfiguration(configuration *api.PodSecurityConfiguration) field.ErrorList {
+const (
+	detailValidLevel   = "must be one of 'privileged', 'baseline', or 'restricted'"
+	detailValidVersion = "must be one of 'latest' or 'v<MAJOR>.<MINOR>'"
```

Comment:
nit: These basically match the error messages returned by `ParseLevel` and `ParseVersion`, so I think you can just use `err.Error()` in place of these strings.